### PR TITLE
update hello-planning-guide.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
clarify Key Infrastructure requirements

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
issue #3351 - clarify Key Infrastructure requirements
issue #1688  - clarify AD requirements for certificate trust
issue #1724  - remove Azure Multi-Factor Authentication as a standalone offering, as this option is not available any more
source: [Azure Multi-Factor Authentication](https://docs.microsoft.com/azure/active-directory/authentication/concept-mfa-howitworks)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ A deployment's trust type defines how each Windows Hello for Business client aut
...
 
-The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the built-in provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers.  Users can authenticate using their certificate to any Windows Server 2008 R2 or later domain controller.
+The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the built-in provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers (but still requires [Windows Server 2016 Active Directory schema](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-prereqs#directories)).  Users can authenticate using their certificate to any Windows Server 2008 R2 or later domain controller.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]:
```suggestion
The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the built-in provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers (but still requires [Windows Server 2016 Active Directory schema](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-prereqs#directories)).  Users can use their certificate to authenticate to any Windows Server 2008 R2, or later, domain controller.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
you recommendations were applied, thank you!

**Comment by mypil** [comment]
@DaniHalfin @mapalko - Can you please assist to check, approve and merge the content for this PR based on issues #1724 #1688 #3351?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin @mapalko @AndreaBarr - I would like to follow up. Can you please assist to check, approve and merge the content for this PR based on issues #1724 #1688 #3351?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? @mapalko has approved the changes already. Thank you.

**Comment by mypil** [comment]
@Justinha @mjcaparas @lizap @DaniHalfin @AndreaBarr - I would like to follow up. Can you please assist to merge the content for this PR based on issues #1724 #1688 #3351?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr @kenwith - would you be able to help merge the content of this PR? @mapalko already approved of the changes and so merging the PR content is what's needed. Thank you.

**Pull request merged**